src/engraving/layout/v0/beamtremololayout.cpp

```diff
@@ -894,10 +894,12 @@ int BeamTremoloLayout::isSlopeConstrained(int startNote, int endNote) const
     // 0 to constrain to flat, 1 to constrain to 0.25, <0 for no constraint
     if (startNote == endNote) {
         return 0;
+    } else if (m_beamType == BeamType::TREMOLO) {
+        return -1;
```

Comment:
isSlopeConstrained is not a good name for this type of method. It should either return bool or have another name